testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -15,7 +17,7 @@ Map apiDocTools = [
     org: 'edx',
     repoName: 'api-doc-tools',
     pythonVersion: '3.5',
-    cronValue: '@weekly',
+    cronValue: '* H(4-11) * * H(0-4)',
```

Comment:
You can extract this variable to a string above this map and then just refer to that string in all the places.
https://groovy-lang.org/syntax.html#all-strings

```
weekly = '* H(4-11) * * H(0-4)'
```

Reply:
Wise.  Done.